code/modules/incon/diaperstatus.dm

```diff
@@ -744,6 +754,10 @@ var/database/db = new("code/modules/incon/InconFlavortextDB.db")
 			set_light(0)
 			REMOVE_TRAIT(src,TRAIT_NOBREATH,INNATE_TRAIT)
 			SEND_SIGNAL(src,COMSIG_CLEAR_MOOD_EVENT,"sanshield")
+		if ("silisuit_hyper")
+			set_light(0)
+			REMOVE_TRAIT(src,TRAIT_NOBREATH,INNATE_TRAIT)
```

Comment:
This line throws me off a bit, I see it was previously there too but it still bothers me.
NOBREATH is what synths have to not breathe, and it is an innate trait, removing it here would break them.
How it ended up being in the file to begin with, I don't get.
This is kind of a mess that's more related to how diapers shouldn't have a ton of buffs applied in the weirdest way... Er, did you test it with synthetics? Can you test it with synthetics and see if it breaks their breathing?

Reply:
I mean... it's literally what's already in the code. If it works on the version currently running then it should work here....

Reply:
And it works because the synths have inherent traits, which are SPECIES_TRAIT, not INNATE_TRAIT